en/publish/publish-vivo-instant-games.md

```diff
@@ -193,6 +187,12 @@ And the rpk package will be generated in the **/build/qgame/dist** directory.
 
 Starting with **v2.1.3**, vivo Mini Games supports subpackage loading, and usage is similar to WeChat Mini Games. Please refer to [Subpackage Loading](../scripting/asset-bundle.md) for details.
 
+## Resource Management for vivo Mini Game Environment
+
+**vivo Mini Game** is similar to **WeChat Mini Game**. There are restrictions on the package size. Resources over **4MB** must be downloaded via a network request.
```

Comment:
Resources over **4MB** must be downloaded via a network request. -> The main package size limit for vivo Mini Game is **4MB**, more than that must be downloaded via a network request.